keyboards/keebio/levinson/keymaps/ksamborski/keymap.c

```diff
@@ -0,0 +1,212 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 3                                                                /* Symbols, Media */
+#define _RAISE 4                                                                /* Numbers, Arrows */
+#define _FUNC  5                                                                /* Function Keys */
+#define _NUMS  6                                                                /* Numpad */
+#define _ADJUST 16
+
+/* short layer aliases */
+#define _QW _QWERTY
+#define _LW _LOWER
+#define _RS _RAISE
+#define _NM _NUMS
+#define _AD _ADJUST
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  MKITPNK,
+  DYNAMIC_MACRO_RANGE
+};
+
+#include "dynamic_macro.h"
```

Comment:
This is depreciated and should be removed. 

```suggestion
```

Instead, you should add `DYNAMIC_MACRO_ENABLE = yes` to your rules.mk, as that enables this feature properly. 

https://docs.qmk.fm/#/feature_dynamic_macros